src/ec_git_vsn.erl

```diff
@@ -15,7 +15,10 @@
 
 %% API
 -export([new/0,
-         vsn/1]).
+         vsn/1,
+	 parse_tags/1,
```

Comment:
if these are internal exports, we may want to have a macro'd out `-ifdef(TEST). -export([...]). -endif.` that makes sure we don't include and exposes these calls from a parent library. This avoids accidentally exposing functions we don't plan on exposing.

Reply:
That's true; I changed it.